### PR TITLE
DDOC-870: Add documentation for MyApps view

### PR DESCRIPTION
**Pull request by bszwarc**
# Description

Added documentation for My Apps
Reorganized and updated documentation for applications

Fixes DDOC-870

### CODE REVIEW

File: content/guides/authorization/custom-app-approval.md [hunk 1]
```diff
@@ -23,6 +23,11 @@ enabled by a Box Admin depending on the [unpublished apps setting][upa].
 A Box Admin needs an application's Client ID in order to properly authorize or
 enable it in the Admin Console. 
 
+<Message>
+You can use[My Apps][apps] view to quickly look up
```
Comment (mgrytsai) [inline content/guides/authorization/custom-app-approval.md:27]: missing space after use

File: content/guides/authorization/custom-app-approval.md
Comment (mgrytsai) [inline content/guides/authorization/custom-app-approval.md:1]: `Scroll down to the OAuth 2.0 Credentials section and copy the Client ID value to provide to a Box Admin.`
Another option is to hover over the app and click on clint id from the My Apps table.

File: content/guides/authorization/custom-app-approval.md (hunk 1 above)
Comment (bszwarc) [reply content/guides/authorization/custom-app-approval.md:27]: Fixed :)

File: content/guides/authorization/custom-app-approval.md
Comment (bszwarc) [reply content/guides/authorization/custom-app-approval.md:1]: Fixed :)

**Review by mgrytsai — APPROVED** [review]
LGTM 🚀

**Pull request merged**